Capfile

```diff
@@ -11,10 +11,10 @@ set :scm, :git
 set :stage, "test" unless exists? :stage
 
 if stage == "production"
-  role :web, "ec2.electionleaflets.org.au"
+  role :web, "electionleaflets.org.au"
```

Comment:
Won't need to change this until after the migration is done because we can leave the ec2 subdomain in place and clear that up later.